crawler_mcp/crawlers/optimized/run.py

```
@@ -0,0 +1,1216 @@
+        locales: list[str] = []
+        for entry in args.allowed_locales:
+            if entry is None:
+                continue
+            e = entry.strip()
+            if not e:
+                continue
+            if e.lower() in ("*", "all"):
+                locales = []
+                break
+            locales.extend([p.strip() for p in e.split(",") if p.strip()])
+        cfg.allowed_locales = [l.lower() for l in locales]
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)